database/datamigrations/2015_01_01_105002_add_world_callingcode.php

```diff
@@ -0,0 +1,1749 @@
+<?php
+
+declare(strict_types=1);
+
+use Carbon\Carbon;
+use Illuminate\Database\Migrations\Migration;
+use Tipoff\Addresses\Models\Country;
+use Tipoff\Addresses\Models\CountryCallingcode;
+
+class AddWorldCallingcode extends Migration
+{
+    public function up()
+    {
+        $callingcodeData =
+            [
+                [
+                    'country_id' => Country::fromAbbreviation('ABW')->getId(),
+                    'code' => '2',
+                    'display' => '+2',
+                    'root' => '+2',
+                    'suffix' => '97',
```

Comment:
The `root` & `suffix` on all of these are correct. The `code` & `display` should include the suffix, so in this case they would be:

```
'code' => '297',
'display' => '+297',
```

The `code` should not contain spaces. The display should be however Wikipedia displays them for each country:

https://en.wikipedia.org/wiki/List_of_country_calling_codes

They include spaces a lot of time when displaying the country calling code. Let's a space in the same spot in the `display` field.